cmd/krel/cmd/push.go

```diff
@@ -152,51 +153,49 @@ func init() {
 }
 
 func runPushBuild(opts *pushBuildOptions) error {
-	log.Println("unimplemented")
-
 	var latest string
 	releaseKind := opts.releaseKind
 
 	// Check if latest build uses bazel
 	dir, err := os.Getwd()
 	if err != nil {
-		log.Fatal(err)
+		return errors.Wrapf(err, "Unable to get working directory: %v")
```

Comment:
```suggestion
		return errors.Wrapf(err, "Unable to get working directory")
```

Reply:
If we are not providing a formatting string then we should likely just use `errors.Wrap()`

Reply:
This makes sense. 👍

Reply:
Existing makes sense or update to `errors.Wrap()`? :)

Reply:
I suggest using o
`Wrap()`. :)